src/convenience/frameworks.shared.ts

```diff
@@ -0,0 +1,48 @@
+// deno-lint-ignore-file no-explicit-any
+
+export const SECRET_HEADER = "X-Telegram-Bot-Api-Secret-Token";
+
+/** express web framework */
+const express = (req: any, res: any) => ({
+    update: Promise.resolve(req.body),
+    header: req.header(SECRET_HEADER),
```

Comment:
Shouldn't it be [request.get](https://expressjs.com/en/5x/api.html#req.get)?

Reply:
It's an alias, isn't it? I think `req.header` is a more accurate term for a request header than `req.get`.

Reply:
It is not anywhere in the documentation, though. Node http which Express extends, the method is `getHeaders` and `headers` is an object.

Reply:
Oh sorry, I didn't read the last line. Nvm lol

Reply:
It's stated at exactly the point in the docs that you linked to.
![Screenshot_20220731-201047](https://user-images.githubusercontent.com/12952387/182039718-2dc631cf-0e53-463a-ae7e-a1e87ac6cb42.png)
↑ here

Reply:
Yeah, sorry for the trouble :)

Reply:
No worries, thanks for reviewing! :)